app/presenters/summary/sections/passporting_benefit_check_partner.rb

```diff
@@ -2,7 +2,7 @@ module Summary
   module Sections
     class PassportingBenefitCheckPartner < Sections::PassportingBenefitCheck
       def show?
-        include_partner_in_means_assessment?
+        super && include_partner_in_means_assessment?
```

Comment:
As this is now an optional question (won't be asked if the partner has an arc), this needs to check if the partner has a benefit type selected

Reply:
Ignore the above, the section is meant to be displayed whether there is a benefit type or not (see designs). Instead I have amended the logic to not display the section if the partner has an arc